lib/discovery.js

```diff
@@ -501,4 +343,12 @@ function mixinDiscovery(DB2, db2) {
     // }
     // return undefined;
   };
+
+  DB2.prototype.setDefaultOptions = function(options) {
```

Comment:
We should have a default implementation in SQLConnector. This way, we don't have to override here.